ui.frontend/src/view/FormFieldBase.js

```diff
@@ -224,6 +225,31 @@ export default class FormFieldBase extends FormField {
         let widgetValue = typeof value === "undefined" ? null :  value;
         if (this.widget) {
             this.widget.value = widgetValue;
+            this.updateEmptyStatus();
+        }
+    }
+
+    /**
+     * updates the html class based on the existence of a value in a field
+     */
+    updateEmptyStatus() {
+        if(!this.getWidget())
+            return;
+
+        const updateModifierClass = (widget, newValue) => {
+            const widgetBemClass = widget.className.split(/\s+/).filter(bemClass => bemClass.endsWith("__widget"))[0];
```

Comment:
`const widgetBemClass = this.bemBlock + '__widget'`

Reply:
@nit23uec `this.bemBlock` is only accessible inside each component's view. This is the base file.